### PR TITLE
Increase VAOS timeout

### PR DESCRIPTION
**Pull request by kreek**
## Description of change
Increases the VAOS (VA Mobile) read timeout to 30 seconds.

## Testing done
unit tests

## Testing planned
staging integration tests

#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by saneshark** [comment]
hopefully you don't need to call &.to_i on the Settings.yml stuff. It can interpret integers in yaml right?

**Comment by edmangimelli** [comment]
>hopefully you don't need to call &.to_i on the Settings.yml stuff. It can interpret integers in yaml right? Also don't want &.to_i, because nil would become 0 and fail the || 30

it seems like I always end up using `Integer()`, but then you need to put it in a `begin/rescue` which is maybe a bit clunky

**Comment by edmangimelli** [comment]
should a default value (30) live somewhere else? just curious if we had some convention

**Comment by kreek** [comment]
@annaswims @edmangimelli created a devops PR that sets the timeout to 30s for staging https://github.com/department-of-veterans-affairs/devops/pull/5920

File: modules/vaos/spec/services/configuration_spec.rb [hunk 1]
```diff
@@ -32,4 +32,10 @@
       end
     end
   end
+
+  describe '#read_timeout' do
+    it 'has a default timeout of 30 seconds' do
```
Comment (annaswims) [inline modules/vaos/spec/services/configuration_spec.rb]:
```suggestion
    it 'has a default timeout of 15 seconds' do
```

**Comment by edmangimelli** [comment]
https://github.com/department-of-veterans-affairs/vets-api/pull/3531#issuecomment-553577976

noice

**Pull request merged**